lib/developer.js

```diff
@@ -78,6 +78,7 @@ function developer (opts) {
     }, opts);
 
     initialRequest(opts)
+      .then(getListWithPermissions(opts))
```

Comment:
Now that you mention it, I figure out that you shouldn't need to add a getListWithPermissions everywhere; the getParseList function should be smart enough to build a function that retrieves any combination of details and permissions according to what the options object specifies. This is not to say they should be coupled like in the first implementation, but that the resulting function should be built dynamically. That may require a bit of refactoring, but I think that's the best way to handle it.